test/new-e2e/tests/installer/windows/remote-host-assertions/remote_windows_host_asserts.go

```diff
@@ -181,3 +181,31 @@ func (r *RemoteWindowsHostAssertions) HasNoNamedPipe(pipeName string) *RemoteWin
 
 	return r
 }
+
+// HasARunningDatadogInstallerService verifies that the Datadog Installer service is installed and correctly configured.
+func (r *RemoteWindowsHostAssertions) HasARunningDatadogInstallerService() *RemoteWindowsHostAssertions {
+	r.suite.T().Helper()
+
+	r.HasAService("Datadog Installer").
```

Comment:
was this copied from `startServiceWithConfigFile` ? could call this there now.

```suggestion
	r.HasAService(installerwindows.ServiceName).
```